circle.yml

```diff
@@ -0,0 +1,12 @@
+machine:
+  xcode:
+    version: 9.1
+  environment:
+    LANG: en_US.UTF-8
+dependencies:
+  override:
+    - pod install --project-directory=Example --repo-update
```

Comment:
If the Pods directory is checked in, you can avoid `pod installing by overriding this with an empty step: https://github.com/RxSwiftCommunity/NSObject-Rx/blob/master/circle.yml#L6